lib/cdo/shared_constants/pd/shared_workshop_constants.rb

```diff
@@ -72,6 +73,18 @@ module SharedWorkshopConstants
         SUBJECT_CSP_FIT = SUBJECT_FIT,
         SUBJECT_CSP_FOR_RETURNING_TEACHERS,
       ],
+      COURSE_CSA => [
+        SUBJECT_CSA_SUMMER_WORKSHOP = SUBJECT_SUMMER_WORKSHOP,
+        SUBJECT_CSA_VIRTUAL_KICKOFF = SUBJECT_VIRTUAL_KICKOFF,
+        SUBJECT_CSA_WORKSHOP_1 = SUBJECT_WORKSHOP_1,
+        SUBJECT_CSA_WORKSHOP_2 = SUBJECT_WORKSHOP_2,
+        SUBJECT_CSA_WORKSHOP_3 = SUBJECT_WORKSHOP_3,
+        SUBJECT_CSA_WORKSHOP_4 = SUBJECT_WORKSHOP_4,
+        SUBJECT_CSA_WORKSHOP_1_2 = SUBJECT_WORKSHOP_1_2,
+        SUBJECT_CSA_WORKSHOP_3_4 = SUBJECT_WORKSHOP_3_4,
+        SUBJECT_CSA_TEACHER_CON = SUBJECT_TEACHER_CON,
+        SUBJECT_CSA_FIT = SUBJECT_FIT,
```

Comment:
FIT shows up here but not in the other places this list shows up. Is that an issue?